x-pack/plugins/ml/public/file_datavisualizer/components/analysis_summary/analysis_summary.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import React from 'react';
+
+import {
+  EuiTitle,
+  EuiSpacer,
+  EuiDescriptionList,
+} from '@elastic/eui';
+
+export function AnalysisSummary({ results }) {
+  const items = createDisplayItems(results);
+
+  return (
+    <React.Fragment>
+      <EuiTitle size="s">
+        <h3>Summary</h3>
+      </EuiTitle>
+
+      <EuiSpacer size="m" />
+
+      <EuiDescriptionList
+        type="column"
+        listItems={items}
+        className="analysis-summary-list"
+      />
+    </React.Fragment>
+  );
+}
+
+function createDisplayItems(results) {
+  const items = [
+    {
+      title: 'Number of lines analyzed',
+      description: results.num_lines_analyzed,
+    },
+    // {
+    //   title: 'Charset',
+    //   description: results.charset,
+    // }
```

Comment:
@jgowdyelastic, could you say whether it is supposed to be removed?
I'm working with that code now and am not sure whether should it be here or not.
I'm writing to you since you are the first contributor to this file.

Reply:
This can be left it, at some point we hope to support the overriding of the file encoding. This was added as a placeholder for that feature.

Reply:
@jgowdyelastic, thanks for the explanation!